src/blocks/pacman.rs

```diff
@@ -172,9 +187,36 @@ fn get_update_count() -> Result<usize> {
     .count())
 }
 
+fn has_kernel_update() -> Result<bool> {
+    let updates_db = get_updates_db_dir()?;
+
+    // check if there are linux kernel updates
+    Ok(String::from_utf8(
```

Comment:
Sorry I forgot to submit this before, so the highlighted code is wrong. But the comments still stand.

Reply:
Sure, that is possible. I tried not to refactor the whole block and make as minimal changes as possible but if it would be okay to refactor the block to be more efficient in resource usage and call of external programs I could do that!

Reply:
I say go for it :+1: